src/components/Api.js

```diff
@@ -0,0 +1,111 @@
+class Api {
+  constructor(options) {
+    this._options = options;
+    this._baseUrl = options.baseUrl;
+    this._authToken = options.headers.authorization;
+  }
+
+  getInitialCards() {
+    return fetch(`${this._baseUrl}/cards`, {
+      headers: {
+        authorization: this._authToken,
```

Comment:
According to the task you need to pass full `headers` into the constructor of `Api`. See here https://snipboard.io/pRdaAZ.jpg . And in the code you need to use only: `headers: this._headers` (not to duplicate the code)